InterpolateColor fails on Android

```
 const  v = useSharedValue(0);
 const animatedStyle = useAnimatedStyle(() => ({
    backgroundColor: interpolateColor(v.value, [-1, 0, 1], ['red', 'blue', 'green']),
  }));
```
fails with the following on Android:

<img width="1311" alt="Screen Shot 2020-11-26 at 10 07 08" src="https://user-images.githubusercontent.com/306134/100330515-57a37d00-2fcf-11eb-8926-e928a529a9cd.png">

wow @wcandillon can you help me out with the reanimated@alpha installation on 64.0-rc
i tried and follow the installation steps on https://docs.swmansion.com/react-native-reanimated/docs/installation but android build fails with output:
```bash
> Task :app:checkDebugAarMetadata FAILED
13 actionable tasks: 2 executed, 11 up-to-date

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Failed to transform react-native-reanimated-64.aar (project :react-native-reanimated) to match attributes {artifactType=android-aar-metadata}.
      > Execution failed for JetifyTransform: C:\Users\hp\Desktop\apps\mobile\rnext\node_modules\react-native-reanimated\android\react-native-reanimated-64.aar.
         > Transform's input file does not exist: C:\Users\hp\Desktop\apps\mobile\rnext\node_modules\react-native-reanimated\android\react-native-reanimated-64.aar. (See https://issuetracker.google.com/issues/158753935)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

Im wondering if you have followed the exact same steps from the installation page or you did some extra.

Please i will appreciate if you can give me some Abracadabra tips.

--- a/src/reanimated2/Colors.js
+++ b/src/reanimated2/Colors.js
@@ -50,8 +50,8 @@ function getMatchers() {
   return cachedMatchers;
 }
 // cachedMatchers is lazy loaded and it is frozen when worklet is being created,
-// is possible to call getMatchers() when object is freezed then cachedMatchers
-// has no assigned regexs
+// it is possible to call getMatchers() when the object is frozen, then cachedMatchers
+// has no assigned regexes
 getMatchers();
 
 function hue2rgb(p, q, t) {

--- a/src/reanimated2/Colors.js
+++ b/src/reanimated2/Colors.js
@@ -49,6 +49,10 @@ function getMatchers() {
   }
   return cachedMatchers;
 }
+// cachedMatchers is lazy loaded and it is frozen when worklet is being created,
+// is possible to call getMatchers() when object is freezed then cachedMatchers
+// has no assigned regexs
+getMatchers();
 
 function hue2rgb(p, q, t) {
   'worklet';

